packages/atomic/package.json

```diff
@@ -24,33 +24,27 @@
     "collection-manifest.json"
   ],
   "scripts": {
```

Comment:
I decided to convert all tasks to `nx:run-commands` tasks instead of `nx:run-scripts` tasks so we can have a clean `package.json` file with only the tasks that are relevant to us while developing.